rosidl_generator_py/rosidl_generator_py/cli.py

```diff
@@ -26,6 +26,14 @@
 
 class GeneratePython(GenerateCommandExtension):
 
+    def __init__(self, name, *, typesupport_implementations=None, **kwargs):
+        super().__init__(name, **kwargs)
```

Comment:
Same comment here about not having `kwargs` here so that typos in the spec lead to exceptions

Reply:
Addressed in https://github.com/ros2/rosidl_python/pull/133/commits/7183fdabfe11febdfa1b8c957988a3446862a09d.